test/src/sdk-harness/Cargo.toml

```diff
@@ -24,3 +23,6 @@ name = "integration_tests"
 path = "test_projects/harness.rs"
 
 [workspace]
+
+[features]
+fuel-core-lib = ["fuels/fuel-core-lib"]
```

Comment:
Could you elaborate a little on why the `sdk-harness` crate needs to expose a `fuel-core-lib` feature here? Currently it appears to be unused by the `sdk-harness` crate.

If we need the `fuels` dependency's `fuel-core-lib` feature in order to be able to successfully build the tests, I think it's fine to just specify this feature directly under the `fuels` dependency (as you've already done). We may not to expose the feature from the crate itself.

Reply:
I think we need to expose it if we want to use "fuel-core-lib" in tests.

Reply:
> I think we need to expose it if we want to use "fuel-core-lib" in tests.

I believe we only need to enable the `fuels` dependency's `fuel-core-lib` (which you've already done). I don't believe there's any need to expose the feature from this crate as we require the feature from `fuels` whether or not the user passes a `--features "fuel-core-lib"` flag.

Could you try removing this `[features]` table?

Reply:
Ditto

Reply:
You're right! I changed this too.